integrations/mage_ai/README.md

```diff
@@ -0,0 +1,107 @@
+# 🧙🏻‍♂️ Machine Learning Pipelines with Mage.ai and Hopsworks
+
+Mage.ai is an open-source platform for orchestrating data and machine learning pipelines. Hopsworks is a ML infrastructure platform. You can run your feature, training, and inference pipelines on Mage.ai and get the benefits of visualization, debugging, and so on.
+
+
+## ⬇️ Installation
+
+
+Create a new Python environment for this tutorial.
+
+Open the terminal and pass the next command:
+
+```
+git clone https://github.com/logicalclocks/hopsworks-tutorials.git
+```
+
+Go to the project folder.
+
+```
+cd hopsworks-tutorials/integrations/mage_ai
+```
+
+Install the mage-ai library:
+
+```
+pip install mage-ai
+```
+
+Install the requirements.txt
+
+```
+pip install -r requirements.txt
+```
+
+To launch the Mage tool run the next command:
+
+```
+mage start mage_tutorial
+```
+
+Open http://localhost:6789 in your browser.
+
+
+## ⚙️ API Key Setup
+
+
+Navigate to the Pipelines section and open the hopsworks_mage_quickstart pipeline.
+Here you can find a Pipeline Tree which shows you the block flow.
+
+![tree](images/tree.png)
+
+Go to the Edit Pipeline section.
+
+![edit](images/edit.png)
+
+
+The next step is to add the Hopsworks Api key to the secrets. Open the Secrets section and press the **+New** button.
+
+![secrets](images/secrets.png)
+
+
+*Secret Name: HOPSWORKS_API_KEY*
+
+*Secret Value: {YOUR_HOPSWORKS_API_KEY}*
+
+![secret](images/secret.png)
+
+Press Enter to save the secret.
+
+![ready_secret](images/ready_secret.png)
+
```

Comment:
Please add this text here:

If you are running your own Hopsworks cluster, you will also have to add the following secrets, where the secret values should be the IP address or FQDN of your Hopsworks cluster and the name of the project in that Hopsworks cluster you are using:

 * HOPSWORKS_HOST
 * HOPSWORKS_PROJECT